apps/prairielearn/src/schemas/infoCourseInstance.ts

```diff
@@ -62,6 +62,11 @@ export const CourseInstanceJsonSchema = z
       )
       .optional()
       .default('Set'),
+    shareSourcePublicly: z
+      .boolean()
+      .describe("If true, the course instance's source code is shared publicly.")
```

Comment:
As noted on Slack recently, the description (at least for assessments) was misleading for an instructor, who thought that this meant that source code solutions would be made available to students. Can we describe better what this actually does? While it's technically true that the source code is shared (read: copyable), the more useful property is that the previews are publicly accessible, and soon one will be able to copy publicly-shared things.

Reply:
how about "If true, the course instance's JSON configuration and question list is available for others to preview and copy"

Reply:
done in dd2e51b894